api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -2202,4 +2204,37 @@ public void deleteChildAccount(final String id) throws UnauthorizedException, Ge
         System.out.println("url: " + url);
         messageBirdService.deleteByID(url, id);
     }
+
+    /**
+     * Returns outbound pricing for the default SMS configuration for the authenticated account.
+     *
+     * @return outbound pricing for the default SMS configuration for the authenticated account
+     *
+     * @throws UnauthorizedException if client is unauthorized
+     * @throws GeneralException      general exception
+     * @throws NotFoundException     if pricing information could not be found
+     *
+     * @see <a href="https://developers.messagebird.com/quickstarts/pricingapi/list-outbound-sms-prices/">Pricing API</a>
+     */
+    public OutboundSmsPriceResponse getOutboundSmsPrices() throws GeneralException, UnauthorizedException, NotFoundException {
```

Comment:
Can you add examples on the examples folder for new endpoints too?  https://github.com/messagebird/java-rest-api/tree/master/examples/src/main/java

Reply:
Done in 1ccb707.